content/03.results.md

```diff
@@ -246,3 +246,28 @@ If the submitter provided cell types, the submitter annotations are compared to
+As described in Materials and Methods, we derived pseudobulk expression matrices from the single-nuclei libraries, and we compared their expression to bulk using a series of linear models (one per ScPCA project) predicting bulk from pseudobulk expression, with a random effect controlling for sample (Figure {@fig:fig6}A, Figure {@fig:figS8}A).
+We observed an expected positive relationship between bulk and pseudobulk expression across all projects.
+
+We next performed an overrepresentation analysis to test if cell type marker genes for our consensus cell types were enriched in either modality, which may in turn indicate that certain cell types were more prevalently detected by either modality.
+To this end, we calculated the per-gene median of each project's model residuals and identified outliers, where positive outliers represent genes that are enriched in bulk RNA-seq, and negative outliers represent genes that are enriched in single-cell/nuclei RNA-seq.
+For each modality, we calculated the odds ratio as: the odds a cell type marker gene is present in the given modality's outliers compared to other genes.
+Following permutation testing and P-value correction to control the FDR at $5%$, we indeed found that several cell type marker gene sets were overrepresented in the bulk modality, but never in the pseudobulk modality (Figure {@fig:fig6}B, Figure {@fig:figS8}B).
+
+In brain and CNS tumors, we nearly exclusively identified stromal (e.g., endothelial and extracellular matrix secreting cells) and/or neuronal cell types (e.g., glial cells and astrocytes), all of which are known to be prevalent non-immune cells in glioma tumor microenvironments [@doi:10.3389/fimmu.2023.1227126; @doi:10.3389/fphar.2024.1355242], as overrepresented in bulk (Figure {@fig:fig6}B).
+The only exception to this was monocytes being overrepresented in bulk for `SCPCP000009` (brain and CNS tumors), which was sequenced at the single-nuclei level while projects `SCPCP000001` (high-grade gliomas) and `SCPCP000002` (low-grade gliomas) were sequenced at the single-cell level.
+This difference may reflect that single-cell approaches have increased sensitivity to detect immune cells relative to single-nuclei approaches [@doi:10.4132/jptm.2022.12.19].
+
+Given that our consensus cell type analysis identified a variety of immune cells from high- and low-grade gliomas (Figure {@fig:fig5}), this analysis suggests that non-immune cells may have been lost during single-cell library preparation.
+Indeed, several of these overrepresented bulk cell types for `SCPCP000001` and `SCPCP000002` were not among the single-nuclei consensus cell types themselves (`SCPCP000001`: "blood vessel endothelial cell", "extracellular matrix secreting cell", "glial cell", "pericyte"; `SCPCP000002`: "blood vessel endothelial cell", "extracellular matrix secreting cell", "microvascular endothelial cell"), further emphasizing loss of these cell types in the single-nuclei data.
```

Comment:
Aren't these projects single-cell?
```suggestion
Indeed, several of these overrepresented bulk cell types for `SCPCP000001` and `SCPCP000002` were not among the single-cell consensus cell types themselves (`SCPCP000001`: "blood vessel endothelial cell", "extracellular matrix secreting cell", "glial cell", "pericyte"; `SCPCP000002`: "blood vessel endothelial cell", "extracellular matrix secreting cell", "microvascular endothelial cell"), further emphasizing the potential loss of these cell types in the single-cell data.
```

Reply:
Oooof yes rough typo. I will generally read over to check if this issue appears elsewhere.